apps/mitt-konto/src/MittKonto/Wrappers/RouteWrapper.purs

```diff
@@ -30,15 +31,14 @@ type SetRouteWrapperState = (State -> State) -> Effect Unit
 class RouteWrapperContent p where
   instantiate :: p -> SetRouteWrapperState -> Effect Unit
 
-component :: forall p. React.Component (Props p)
-component = React.createComponent "RouteWrapper"
-
-routeWrapper :: forall p. (RouteWrapperContent p) => (Props p) -> JSX
-routeWrapper = make component
-  { initialState
-  , didMount
-  , render
-  }
+routeWrapper :: forall p t. Eq t => RouteWrapperContent p => (p -> t) -> Component (Props p)
```

Comment:
:thinking: I don't get why we need to pass a function now?

Reply:
Because RouteWrapper's rendering is done inside the `useEffect` in this version and `useEffect` is only called if its first parameter has changed value. That function is used to build the value that `useEffect` uses as its trigger condition.